# HG changeset patch
# User Marco Caliari <marco.caliari@univr.it>
# Date 1523175175 -7200
#      Sun Apr 08 10:12:55 2018 +0200
# Branch stable
# Node ID b8279dd836649140324ad143882e87a6f39ffd11
# Parent  57f150c4fe6dbe246643572141fa50c7e9263faa
Turn off warning about sparse lu factorization in bicg BIST test (bug #53390).

* bicg.m: Turn off warning when using a two-output sparse lu factorization.

diff --git a/scripts/sparse/bicg.m b/scripts/sparse/bicg.m
--- a/scripts/sparse/bicg.m
+++ b/scripts/sparse/bicg.m
@@ -531,13 +531,14 @@ endfunction
 %! M = magic (5);
 %! [x, flag, relres] = bicg (A, b, [], 2, M);
 %! assert (norm (b - A * x) / norm (b), 0, relres);
 
 ## Preconditioned technique
 %!testif HAVE_UMFPACK
 %! A = sparse (toeplitz ([2, 1, 0, 0, 0], [2, -1, 0, 0, 0]));
 %! b = sum (A, 2);
+%! warning ("off", "Octave:lu:sparse_input", "local");
 %! [M1, M2] = lu (A + eye (5));
 %! [x, flag] = bicg (A, b, [], 1, M1, M2);
 %! ## b has two columns!
 %! [y, flag]  = bicg (M1 \ A / M2, [M1 \ b, M2' \ b], [], 1);
 %! assert (x, M2 \ y, 8 * eps);
